CONTRIBUTING.md

```diff
@@ -34,12 +34,31 @@ Contributions via pull requests are much appreciated. Before sending us a pull r
 
 To send us a pull request, please:
 
-1. Fork the repository.
+1. Fork the repository. (Note, core members can directly push to a branch on the repository)
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
 3. Ensure local tests pass.
-4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request interface.
-6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+4. Pull request title and message must adhere to [Conventional Commits](https://www.conventionalcommits.org). This means that the title must begin with `feat(scope): title`, `fix(scope): title`, `docs(scope): title`, `refactor(scope): title` or `chore(scope): title`. Including `(scope)` is optional.
```

Comment:
A PR may do more than one thing, and we should explain this. For example, `feat` should include `docs`.